src/getting-started/contribute.md

```diff
@@ -46,21 +48,23 @@ When you see a problem with one of our pages, please let us know. This could be
 
 ## Submit content changes
 
-All of our content resides on GitHub, a well-known source control platform. Anyone can [create a free GitHub account][2] and make content changes and additions through the GitHub web UI. If you are a developer or know your way around git source control and local build environments, you can choose to [work with our source files that way][3].
+All of our content resides on GitHub, a well-known source control platform. Anyone can [create a free GitHub account][2] and make content changes and additions through the GitHub web UI. If you are a developer or know your way around Git source control and local build environments, you can choose to [work with our source files that way][3].
 
-For content contributions to be accepted and published, whether large or small, you must sign the [Adobe Contributor License Agreement (CLA)][4]. This is then associated with your GitHub account, and you do not need to do it again.
+For content contributions to be accepted and published, whether large or small, you must sign the [Adobe Contributor License Agreement (CLA)][4]. This CLA is associated with your GitHub account. You will not need to sign it again.
 
 ### Provide a quick edit
 
-Making quick edits is a good way to fix small errors and omissions in the User Guide. If a page displays an edit link (as shown below), you can make a quick fix yourself. When you edit the document, you submit a pull request (PR) to submit the fix/suggestion to us. Then we can review, approve, and publish the submitted change.
+Making quick edits is a good way to fix small errors and omissions in the User Guide. If a page displays an edit link (as shown below), you can make a quick fix yourself. When you edit the document, you send us a pull request (PR) to submit the fix/suggestion to us. Then we can review, approve, and publish the submitted change.
+
+To make a quick edit:
 
-1. Open the User Guide page where you want to make an edit or update.
+1. Open the User Guide topic you want to edit or update.
```

Comment:
For external reader I think page makes more sense than topic. I know we use topic as docs people, but it could have other meanings for some.

Reply:
Ok. Topics is what we use on all the other doc sites (at least as far as I know), so again, I hadn't seen this approach.